Event-Management-Application/event_management/src/main/java/com/example/event_management/controller/EventsController.java

```diff
@@ -0,0 +1,63 @@
+package com.example.event_management.controller;
+
+import com.example.event_management.model.BookEventRequest;
+import com.example.event_management.model.BookEvents;
+import com.example.event_management.model.Events;
+import com.example.event_management.service.EventsService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "api/events")
+@CrossOrigin(origins = "https://localhost:3000")
+public class EventsController {
+
+    @Autowired
+    private EventsService eventsService;
+
+    //for admin and user
+    @GetMapping(path = "")
+    public List<Events> retrieveAllEventsList() {
+        return eventsService.findAll();
+    }
+
+    //for admin
+    @GetMapping(path = "/{id}")
+    public Events retrieveEventById(@PathVariable String id){
+        return eventsService.findById(id);
+    }
+
+//    //for admin and user
+//    @GetMapping(path = "/event-name/{eventName}")
+//    public Events retrieveEventByEventName(@PathVariable String eventName){
+//        return eventsService.findByName(eventName);
+//    }
+
+    //for admin
+    @PostMapping(path = "/add-event")

```

Comment:
according to REST API standards.
```suggestion
    @PostMapping
```